frontend/src/components/PackageGeneration/Editor.js

```diff
@@ -35,9 +35,31 @@ class editor extends React.Component {
     }
 
    componentDidMount() {
-    this.setState({code: localStorage.getItem("packageConfigYAML")})
-    this.setState({title: JSON.parse(localStorage.getItem("packageConfigJSON"))["bundle"]["title"]})
-    this.setState({description: JSON.parse(localStorage.getItem("packageConfigJSON"))["bundle"]["description"]})
+
+    if (localStorage.getItem("packageConfigYAML")) {
+      this.setState({code: localStorage.getItem("packageConfigYAML")})
+    } else {
+      this.setState({code: "Kindly generate or paste your configuration here"})
+    }
+
+    if (!localStorage.getItem("packageConfigJSON")["bundle"] === undefined ) {
+      if (localStorage.getItem("packageConfigJSON")["bundle"]["title"] === undefined ) {
+        this.setState({title: "No title Specified"})
```

Comment:
What do you expect the user to do under this condition?

Reply:
The user cannot do anything. He cannot manually change the title unless he goes back and creates a new configuration